app/models/chargeback/consumption_with_rollups.rb

```diff
@@ -129,6 +129,14 @@ def metering_used_fields_present
       @metering_used_fields_present ||= @rollup_array.count { |rollup| metering_used_fields_present?(rollup) }
     end
 
+    def metering_allocated_for(metric)
```

Comment:
I'm not that familiar with chargeback reports...

But this looks like a rollup.
Is there a reason to not just do a `count(:column)` in sql?
I guess the non-zero clause could be a little tricky.

Reply:
it is rollup, but after method `pluck` so there are just values.

Reply:
darn. especially since these all look like sql `group by` rollups.

GL